action-extension/shopify.extension.toml.liquid

```diff
@@ -1,4 +1,4 @@
-api_version = "unstable"
+api_version = "2023-07"
```

Comment:
We need to update the Host to serve the latest set of components to this version.